### PR TITLE
Changed card content of blockchain dev path

### PR DESCRIPTION
**Pull request by pranavvdesai**
## Related Issue 

- Changed card content on blockchain dev path

Closes: #46 

### Describe the changes you've made

My branch wasn't updated with the responsive changes made, reverted and made changes.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

image

 
## Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -107,21 +107,22 @@ <h1 class="w-full my-2 text-4xl font-bold leading-tight text-center text-gray-80
 
     <div class="text-black grid grid-col-1 p-3 gap-y-3 md:grid md:grid-cols-2 md:gap-3 md:p-3">
 
-      <!--Framer-->
+      <!--Simplilearn-->
       <div
         class="cursor-pointer  bg-white flex-1 rounded-md shadow focus:outline-none focus:shadow-outline transform transition hover:scale-105 hover:shadow-lg duration-300 ease-in-out p-4">
         <div class="flex items-center justify-start overflow-hidden">
-          <img alt="framer" class="bg-gray-50 p-5 rounded-md" src="./static/img/framer.png" />
+          <img alt="framer" class="bg-gray-50 p-5 rounded-md" src="./static/img/simplilearn.png" />
```
Comment (shuklaritvik06) [inline src/index.html:114]: Please change alt from framer to simplilearn

File: src/index.html [hunk 2]
```diff
@@ -131,20 +132,21 @@ <h4 id="name" class="text-xl font-semibold">Framer</h4>
         </div>
       </div>
 
-      <!--Whimsical-->
+      <!--Learn with Whiteboard-->
       <div
         class="cursor-pointer bg-white flex-1 rounded-md shadow focus:outline-none focus:shadow-outline transform transition hover:shadow-lg hover:scale-105 duration-300 ease-in-out p-4">
         <div class="flex items-center justify-start overflow-hidden">
-          <img alt="whimsical" class="bg-gray-50 p-5 rounded-md" src="./static/img/whimsical.png" />
+          <img alt="whimsical" class="bg-gray-50 p-5 rounded-md" src="./static/img/whiteboard.png" />
```
Comment (shuklaritvik06) [inline src/index.html:139]: same change alt here

File: src/index.html [hunk 3]
```diff
@@ -154,20 +156,21 @@ <h4 id="name" class="text-xl font-semibold">Whimsical</h4>
         </div>
       </div>
 
-      <!--MindMaster-->
+      <!--Figment-->
       <div
         class="cursor-pointer bg-white flex-1 rounded-md shadow focus:outline-none focus:shadow-outline transform transition hover:scale-105 hover:shadow-lg duration-300 ease-in-out p-4">
         <div class="flex items-center justify-start">
-          <img alt="mindmaster" class="bg-gray-50 p-5 rounded-md" src="./static/img/mindmaster.png" />
+          <img alt="mindmaster" class="bg-gray-50 p-5 rounded-md" src="./static/img/figment.png" />
```
Comment (shuklaritvik06) [inline src/index.html:163]: same here

File: src/index.html [hunk 4]
```diff
@@ -177,19 +180,21 @@ <h4 id="name" class="text-xl font-semibold">MindMaster</h4>
         </div>
       </div>
 
-      <!--Zapier-->
+      <!--CryptoZombies-->
       <div
         class="flex-1 cursor-pointer bg-white flex-1 rounded-md shadow focus:outline-none focus:shadow-outline transform transition hover:shadow-lg hover:scale-105 duration-300 ease-in-out p-4">
         <div class="flex items-center justify-start">
-          <img alt="zapier" class="bg-gray-50 p-5 rounded-md" src="./static/img/zapier.png" />
+          <img alt="zapier" class="bg-gray-50 p-5 rounded-md" src="./static/img/cryptozombie.png" />
```
Comment (shuklaritvik06) [inline src/index.html:187]: same here

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments by @shuklaritvik06.

**Review by Panquesito7 — APPROVED** [review]
Awesome! Great work! Thanks. 🚀

**Review by shuklaritvik06 — APPROVED** [review]
LGTM :)

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Comment by KukretiShubham** [comment]
Thanks for the contribution

**Review by mkubdev — COMMENTED** [review]
Thanks for your contribution

**Pull request merged**